packages/flutter_test/lib/src/controller.dart

```diff
@@ -1243,6 +1458,8 @@ abstract class WidgetController {
   /// the widget's render object has been laid out at least once.
   Rect getRect(Finder finder) => Rect.fromPoints(getTopLeft(finder), getBottomRight(finder));
 
+  // TODO(pdblasi-google): Deprecate this and point references to semantics.find. See [112670](https://github.com/flutter/flutter/issues/112670).
```

Comment:
Since this is not a doc string, the markdown won't work. just inline the link would be fine. Also may be move right above the API below the docstring.

Reply:
Oh neat! Good to know! If it's not the `///` doc comment, it won't be included in the docs even if it's "connected"?

Reply:
no, it won't be included i think.